azdev/utilities/path.py

```diff
@@ -190,7 +197,11 @@ def get_path_table(include_only=None, include_whl_extensions=False):
         }
     }
     """
-    from azure.cli.core.extension import EXTENSIONS_DIR  # pylint: disable=import-error
+    config = get_azure_config()  # pylint: disable=import-error
+    try:
+        EXTENSIONS_DIR = config.get(const.EXT_SECTION, const.AZ_DEV_SRC)
+    except (configparser.NoSectionError, configparser.NoOptionError):
```

Comment:
The error will never raise as far as I know (it might be a bug, raising error is expected). 
So, it will end up crashing at line 214 as it's an empty string if setup without -r.

If setup without -r, the content of `~/.azure/config` would like:
```
[cloud]
name = AzureCloud

[extension]
dev_sources = 

[core]
cache_ttl = 10
first_run = yes
collect_telemetry = no
```

Reply:
Installing without -r will cause multiple azdev functions to essentially be useless. (azdev extension add, list, build, create, etc).  What use cases do we want to support were a user installs without -r?

Reply:
Also, this was already unstable in the old version. It just set this to "_NONE_".  This could trick users in to thinking they have correctly setup azdev correctly when they have not.

Reply:
Added support to install the new way without -r